### ISSUE TITLE
TeamCity agent image (windows) doesn't work in azure

### ISSUE THREAD

**Issue by dreschner** [JetBrains/teamcity-azure-agent#113]
## Description

I try to use the teamcity windows agent docker images within azure. Unfortunately, that doesn't work as expected. As you can see in the agent log, some operations to generate the configuration file fails.

I guess that's because there is no storage account attached to the container. But the UI only let my attach a storage account on the linux image (which starts and connects fine).

If you need any more data, please let me know! Thank you very much for your help.

## Environment

* TeamCity version: 2019.1
* Azure plugin version: 0.8.11

## Diagnostic logs
[agent-log.txt](https://github.com/JetBrains/teamcity-azure-agent/files/3273352/agent-log.txt)
![image-settings](https://user-images.githubusercontent.com/34408442/59219186-24461600-8bc2-11e9-8251-54265196f103.png)
![container-properties](https://user-images.githubusercontent.com/34408442/59219185-23ad7f80-8bc2-11e9-8f44-01d6358683f1.png)

**Comment by davidmine** [comment]
I'm having similar problems with a windows agent. I'm running windows docker container within a Teamcity server deployed on Azure. The agent starts, but never registers any runners or parameters. I get ![this error](https://user-images.githubusercontent.com/2338943/62374851-db984280-b50a-11e9-82a3-ee649642d84d.jpg), but I don't believe it. If the agent didn't have the server-url properly configured, how would the server know to display the agent? My best guess is that some other configuration setting is causing the launch to break and that's why the agent has no listed runners

I'm able to pull the docker image to my local machine, run the container, and have the agent successfully connect to the server deployed on Azure. I know that the docker image works without any modification.

My next thought was that the Teamcity server version was the problem. I was able to successfully update the "Deploy to Azure" server instance to 2019.2.1 (including a few VM reboots for good measure), but that didn't solve the problem either. At this point, my logs are pretty jumbled between all the stuff that I tried, but I would be happy to delete everything, start again with a specific test, and upload those logs.

Any update on this?
